server/auth.js

```diff
@@ -15,7 +15,7 @@ exports.login = async function (username, password) {
         return null;
     }
 
-    let user = await R.findOne("user", " username = ? AND active = 1 ", [
+    let user = await R.findOne("user", " username LIKE ? AND active = 1 ", [
```

Comment:
Thanks, I didn't notice that, because I thought SQLite is similar MySQL and MySQL is case-insensitive by default.

However, I don't think using `LIKE` is a best choice.

Reply:
Yes I agree, using LIKE seems a bit off.
I am mostly used to Entity Framework where we would use `.Where(u => u.username.ToLower == username.ToLower())`

After some investigation it seems like setting `COLLATE NOCASE` on the username property itself is more reasonable. I created a commit which introduces a patch sql file dependent on the two 2fa sql patches.
- Doing this requires recreating the SQLITE table as ALTER TABLE can not modify existing columns. I am not completely sure if this has other side effects